src/components/clicker.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+class Clicker extends React.Component {
+  constructor() {
+    super();
+    this.state = { count: 0, currentCount: 1, enoughPoints: " " };
+  }
+  handleAddClick = () => {
+    const { count, currentCount } = this.state;
+    this.setState({
+      count: count + currentCount,
+      enoughPoints: " "
```

Comment:
if enough state is gonna be set to either true or false, the `enoughPoints` state should start as a boolean